tests/screenshots/cases/software/basic/rgb.slint

```diff
@@ -0,0 +1,13 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+TestCase := Window {
+    width: 64px;
```

Comment:
What's the default if you don't specify the size?

Reply:
0,0